### PR TITLE
DOC: df.to_html documentation incorrectly contains min_rows optional param (release note)

### LINKED ISSUE #44304: DOC: df.to_html documentation incorrectly contains min_rows optional param
### 

- [X] I have checked that the issue still exists on the latest versions of the docs on `master` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_html.html

### Documentation problem

The documentation contains an optional parameter `min_rows`, but the source code shows that no option is supported. See source code [here](https://github.com/pandas-dev/pandas/blob/v1.3.4/pandas/core/frame.py#L2711).

Verify this optional param is not supported with the following code:
```
import pandas as pd
df = pd.DataFrame({'data': [1, 2, 3]})
df.to_html(max_rows=2, min_rows=1)
```
Which outputs the error
```
      1 import pandas as pd
      2 df = pd.DataFrame({'data': [1, 2, 3]})
----> 3 df.to_html(max_rows=2, min_rows=1)

TypeError: to_html() got an unexpected keyword argument 'min_rows'
```

### Suggested fix for documentation

Remove the `min_rows` optional param from this documentation.

### PR DESCRIPTION
**Pull request by loicdiridollou**
- [x] closes #44304 
- [x] tests added / passed
- [x] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [x] whatsnew entry

### CODE REVIEW

File: doc/source/whatsnew/v1.4.0.rst [hunk 1]
```diff
@@ -212,6 +212,7 @@ Other enhancements
 - :meth:`.GroupBy.mean` now supports `Numba <http://numba.pydata.org/>`_ execution with the ``engine`` keyword (:issue:`43731`)
 - :meth:`Timestamp.isoformat`, now handles the ``timespec`` argument from the base :class:``datetime`` class (:issue:`26131`)
 - :meth:`NaT.to_numpy` ``dtype`` argument is now respected, so ``np.timedelta64`` can be returned (:issue:`44460`)
+- Change in the order of the arguments in :meth:`DataFrame.to_string` due to change in the docstring (:issue:`44304`)
```
Comment (mroeschke) [inline doc/source/whatsnew/v1.4.0.rst:215]: This should go under `Other API changes` and mention that `min_rows` was affected.

**Review by mroeschke — APPROVED** [review]

File: doc/source/whatsnew/v1.4.0.rst (hunk 1 above)
Comment (loicdiridollou) [reply doc/source/whatsnew/v1.4.0.rst]: You are correct! Updated it, waiting for tests to run, surprisingly some workflows either got cancelled or failed for no obvious reasons, will look into it

Comment (loicdiridollou) [reply doc/source/whatsnew/v1.4.0.rst]: Some tests are breaking but it is not related to my code, have seen these failed in other PRs 
```
FAILED pandas/tests/plotting/test_hist_method.py::TestDataFrameGroupByPlots::test_axis_share_x
FAILED pandas/tests/plotting/test_hist_method.py::TestDataFrameGroupByPlots::test_axis_share_y
FAILED pandas/tests/plotting/test_hist_method.py::TestDataFrameGroupByPlots::test_axis_share_xy
FAILED pandas/tests/plotting/test_series.py::TestSeriesPlots::test_area_sharey_dont_overwrite
FAILED pandas/tests/plotting/frame/test_frame.py::TestDataFramePlots::test_area_sharey_dont_overwrite
FAILED pandas/tests/plotting/frame/test_hist_box_by.py::TestHistWithBy::test_axis_share_x_with_by
FAILED pandas/tests/plotting/frame/test_hist_box_by.py::TestHistWithBy::test_axis_share_y_with_by
```

Comment (loicdiridollou) [reply doc/source/whatsnew/v1.4.0.rst]: Bugs in the CI Docs and web related to #44494, no idea why but all the recent PRs have this error

**Comment by loicdiridollou** [comment]
Mentioning the failed tests in the following issue #44521

File: doc/source/whatsnew/v0.9.1.rst [hunk 2]
```diff
@@ -59,7 +59,7 @@ New features
 
         .. ipython:: python
 
-    	    df = DataFrame(np.random.randn(5, 3), columns = ['A','B','C'])
+    	    df = pd.DataFrame(np.random.randn(5, 3), columns = ['A','B','C'])
```
Comment (mroeschke) [inline doc/source/whatsnew/v0.9.1.rst:62]: Please revert this change (the CI is having issue unrelated to this PR)

Comment (loicdiridollou) [reply doc/source/whatsnew/v0.9.1.rst]: It was raising a warning on my machine when building but does not seem to be the case on the builder in GH so reverted

**Comment by simonjayhawkins** [comment]
Thanks @loicdiridollou

**Pull request merged**